src/test/java/net/openhft/chronicle/bytes/Bytes4Test.java

```diff
@@ -22,7 +22,11 @@ public void segFault() {
         BytesStore bs = BytesStore.from("this is a test          ");
         bs.append(14, 16, 12);
         bs.isClear();
-        bs.writeUtf8(14, "this is a another text it should over write the other");
+        try {
+            bs.writeUtf8(14, "this is a another text it should over write the other");
+        } catch (Throwable throwable) {
+            fail(throwable.getMessage());
```

Comment:
why do this, if the test throws an exception it will fail the test anyway

Reply:
yes